mono/hello.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+public class HelloWorld
+{
+    static public void Main ()
+    {
+        Console.WriteLine ("Hello Mono World");
```

Comment:
@treeder where are our .Net libs? don't see anything on GH and this code taken from http://dev.iron.io/worker/languages/dotnet/#payload_example